docs/ng-cli-webpack.adoc

```diff
@@ -90,10 +90,10 @@ Next we will modify the application index HTML file to load the web components p
   ...
 
   <script src="bower_components/webcomponentsjs/webcomponents-lite.min.js"></script>
-  <link rel="import" href="bower_components/paper-styles/color.html">
-  <link rel="import" href="bower_components/paper-styles/typography.html">
-  <link rel="import" href="bower_components/vaadin-combo-box/vaadin-combo-box.html">
-  <link rel="import" href="bower_components/paper-input/paper-input.html">
+  <link rel="import" href="/assets/bower_components/paper-styles/color.html">
+  <link rel="import" href="/assets/bower_components/paper-styles/typography.html">
+  <link rel="import" href="/assets/bower_components/vaadin-combo-box/vaadin-combo-box.html">
+  <link rel="import" href="/assets/jjjjbower_components/paper-input/paper-input.html">
```

Comment:
Soooo sorry @oravecz !! My notebook keyboard is awfull, repeting the letter "j" every time haha. I will fix this :)